lms/djangoapps/verify_student/management/commands/tests/test_manual_verify_student.py

```diff
@@ -0,0 +1,55 @@
+"""
+Tests for django admin commands in the verify_student module
+
+"""
+
+from django.core.management import call_command
+from django.test import TestCase
+from nose.tools import assert_equals
+from lms.djangoapps.verify_student.models import ManualVerification
+from lms.djangoapps.verify_student.utils import earliest_allowed_verification_date
+from student.tests.factories import UserFactory
+
+
+class TestVerifyStudentCommand(TestCase):
+    """
+    Tests for django admin commands in the verify_student module
+    """
+
+    def setUp(self):
+        super(TestVerifyStudentCommand, self).setUp()
+        self.user1 = UserFactory.create()
+        self.user2 = UserFactory.create()
+
+    def test_manual_verifications(self):
+        """
+        Tests that the manual_verifications management command executes successfully
+        """
+        assert_equals(len(ManualVerification.objects.filter(status="approved")), 0)
+
+        call_command('manual_verifications', '--email-id', self.user1.email, self.user2.email)
+
+        assert_equals(len(ManualVerification.objects.filter(status="approved")), 2)
```

Comment:
nit: `assertEqual`, we use `assertEquals` for strings, also we can use `count` method for finding the number of objects.
```python
assert_equals(ManualVerification.objects.filter(status='approved').count(), 2)
```